CalibCalorimetry/HcalAlgos/src/HcalPulseContainmentAlgo.cc

```diff
@@ -48,26 +54,30 @@ void HcalPulseContainmentAlgo::init(int num_samples) {
 #if 0
     char s[80];
     sprintf (s, "%7.3f %8.5f %8.5f\n", tmin, bin0val, bin1val);
-    cout << s;
+    std::cout << s << std::endl;
```

Comment:
My understanding is that the use of std::cout is discouraged. Please use the relevant MessageLogger instead, https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideMessageLogger